TFUtil.py

```diff
@@ -8651,16 +8651,26 @@ def get_positional_encoding(num_channels, length=None, position=None, min_timesc
     position = tf.range(length)
   else:
     assert length is None
-    position.set_shape((None,))
+    if len(position.shape) == 1:
+      position.set_shape((None,))
```

Comment:
This does not make sense. This is redundant.

Reply:
Removed.